ethdb/pebble/pebble.go

```diff
@@ -338,6 +338,17 @@ func (d *Database) Delete(key []byte) error {
 	return d.db.Delete(key, nil)
 }
 
+// DeleteRange deletes all of the keys (and values) in the range [start,end)
+// (inclusive on start, exclusive on end).
+func (d *Database) DeleteRange(start, end []byte) error {
+	d.quitLock.RLock()
+	defer d.quitLock.RUnlock()
+	if d.closed {
+		return pebble.ErrClosed
+	}
+	return d.db.DeleteRange(start, end, nil)
```

Comment:
please use the writeOption `return d.db.DeleteRange(start, end, d.writeOptions)`

and please also fix it Delete as well

Reply:
done